src/test/groovy/org/shipkit/internal/gradle/ReleaseConfigurationGettersAndSettersTest.groovy

```diff
@@ -0,0 +1,61 @@
+package org.shipkit.internal.gradle
+
+import org.shipkit.gradle.ReleaseConfiguration
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Unroll
+import testutil.ReflectionUtil
+
+import java.lang.reflect.Method
+
+class ReleaseConfigurationGettersAndSettersTest extends Specification {
```

Comment:
@szczepiq I changed description of PR. It is enough? Or I should add something more?

Thanks for review:)